pkg/multitenant/mtinfopb/info.proto

```diff
@@ -62,9 +69,30 @@ message ProtoInfo {
     (gogoproto.nullable) = false
   ];
 
-  // Next ID: 7
+  // If this tenant was previous a physical replication target that
+  // was promoted. This is the timestamp at which it was promoted.
+  optional PreviousSourceTenant previous_source_tenant = 7 [(gogoproto.nullable) = false];
```

Comment:
I'm probably thinking about this the wrong way but `PreviousSourceTenant` to me implies that this is information about when this tenant was the source of the replication stream. The comment however talks about this being the target (i.e. destination) and when it was promoted to be the primary after cutover. I think in this case the comment is correct but the name is a little confusing?

Reply:
Right, the tenant the overall ProtoInfo describes used to be a destination tenant whose source is the cluster identified by PreviousSourceTenant.  Given that, any ideas about a better name?

Reply:
(I think this name will make more sense in my next commit that adds the ClusterID and TenantID of the previous source cluster.

Reply:
the new comment makes this much clearer, I also see it clearer now when thinking about it in the context of the broader ProtoInfo.